docs/machine-learning/how-to-guides/serve-model-web-api-ml-net.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Serve Machine Learning Model In ASP.NET Core Web API
+description: Serve ML.NET sentiment analysis machine learning model over the internet using ASP.NET Core Web API
+ms.date: 03/05/2019
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use my ML.NET Machine Learning model through the internet using an ASP.NET Core Web API
+---
+
+# How-To: Serve Machine Learning Model Through ASP.NET Core Web API
+
+This how-to shows how to serve a pre-built ML.NET machine learning model to the web using an ASP.NET Core Web API. By doing so it allows for users to access the API for prediction purposes via standard HTTP methods.
+
+## Prerequisites
+
+- [Visual Studio 2017 15.6 or later](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017) with the ".NET Core cross-platform development" workload installed.
+- Powershell.
+- Pre-trained model.
+    - Use the [ML.NET Sentiment Analysis tutorial](../tutorials/sentiment-analysis.md) to build your own model.
+    - Download this [pre-trained sentiment analysis machine learning model](../../../samples/machine-learning/models/sentimentanalysis/sentiment_model.zip)
```

Comment:
This needs to link to the GitHub repo: https://github.com/dotnet/samples/blob/master/machine-learning/models/sentimentanalysis/sentiment_model.zip

Reply:
Resolved